libraries/docker/steps/build.groovy

```diff
@@ -11,7 +11,7 @@ void call(){
       if (config.path_dockerfile != "") {
         sh """
           export base_images=`cat $config.path_dockerfile | grep FROM | awk '{print \$2}'`
-          for i in \$base_images; do docker pull \$i; done;
+          for i in \$base_images; do docker pull \$i || true; done;
```

Comment:
How about checking base_images and skipping if the image exists?

```
export base_images=$(cat "${config.path_dockerfile}" | grep FROM | awk '{print \$2}')
if [[ -n "$(docker images -q ${base_image}" ]]; then
  for i in \$base_images; do docker pull \$i; done;
fi
```

Reply:
because `docker images -q` will return the ID of local image, we should invert the logic inside condition check by `!`